.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -7,5 +7,5 @@ contact_links:
     url: https://github.com/openjs-foundation/cross-project-council#help-with-legal-questions-for-existing-projects-only
     about: "Do you need help with legal matters concerning an OpenJSF project?"
   - name: Infrastructure Request
-    url: https://github.com/openjs-foundation/cross-project-council#help-with-project-infrastructure
+    url: https://github.com/openjs-foundation/cross-project-council#help-with-lf-it-supported-infrastructure
```

Comment:
can we restore the dead link instead?

Reply:
@ljharb added a `<div />` to the markdown file with an `id` attribute. Wdyt?

Reply:
IMO seems unscalable to add empty divs to the file every time the name of a markdown heading changes.

Makes more sense to just update the link or revert the heading, but I'm assuming we made that change for a reason. 🤔